src/QEQ/fix_qeq.cpp

```diff
@@ -777,7 +790,7 @@ void FixQEq::read_file(char *file)
         auto values = reader.next_values(0);
 
         if (values.count() == 0) continue;
-        if (values.count() < 6)
+        if (values.count() < 9)
```

Comment:
We want to maintain backwards compatibility, so these changes to the `read_file()` function needs to be moved to the child class so the parent class is unchanged.